README.md

```diff
@@ -12,6 +12,9 @@ Overview
 
 **React Intl is part of [FormatJS][].** It provides bindings to React via its components and API.
 
+**New:** Join us on Slack at [react-intl.slack.com](https://react-intl.slack.com/) for help, general conversation and more 💬🎊🎉
```

Comment:
nit: instead of saying "New", can it say "Slack" just because eventually it won't be New anymore.